### ISSUE TITLE
Adding code of conduct, licensing, automations

### ISSUE THREAD

**Issue by ChristinaLast** [AQ-AI/openaq-engine#14]
## Key areas to consider 
- [x] Automations in Slack or Email to ensure that you are notified in the best way possible if a new issue/bug/new comment.
https://slack.github.com/ will let you connect Github to Slack and you can subscribe to all repo events like Issues raised, new comments etc and never miss a message
- I would suggest you separate notifications that are from users / community like new Issues from the PRs (needs a response in the Issue vs PR needs review) as it can get really noisy and easy to miss conversations with people
- [x] create issue template
- Driving consensus and accept compromise as well as conflict resolution and management
- [x] adding code of conduct 
All open sourced repos should have:
- [ ] [LICENSE](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)
- [x] README
- [x] [CONTRIBUTOR](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) guidelines 
- [x] Cover how someone can contribute
- [x] Cover when someone should reach out to you before starting work
- [x] Cover how to open a PR/what must be present in the PR (aka tests) before you will consider merging it. Make it easy for someone to contribute code to the standard and style you expect
- [x] [ISSUE](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) templates (for[ bugs, vs feature requests](https://github.com/snyk-tech-services/snyk-api-import/tree/master/.github/ISSUE_TEMPLATE)) 
- [x] Code of conduct, see Django repo for example 
- [x] [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file
- [x] Your `main` / `master` and `develop` branch must [be protected](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches), and require a - [x] Pull Request review from your own team + all required checks must pass. This will prevent someone being able to raise a PR and merge it if they have repo access unless approved. So for security reasons mainly and to prevent any unexpected merges.
- [ ] Required checks on a branch: tests CI, any linting, security CI etc. What do you use today?
- [ ] All of the above files linked in the README for easy access
- [x] Versioning and releases of packages
- [ ] Instead of manual release steps follow this [setup guide for publishing via Github Actions on Pypi](https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/) as well as [semantic release versioning package for Python](https://python-semantic-release.readthedocs.io/en/latest/index.html) (Python uses semantic release for version bumps) and they have a [specific guide for using Github actions](https://python-semantic-release.readthedocs.io/en/latest/automatic-releases/github-actions.html#).
If this is aimed to be consumed by the Open Source community and the package itself is a dependency in another package, versioning will be key to keep projects compatible and working well together. You can read some information on how[ to version and release here](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/). Generally the package once published will be available on [PyPi](https://towardsdatascience.com/how-to-publish-a-python-package-to-pypi-7be9dd5d6dcd) for anyone to find and use and you can consume it inside your other projects as a dependency at a specific version.
